src/schemas/json/typedoc.json

```diff
@@ -0,0 +1,141 @@
+    },
+    "excludeExternals": {
+      "default": false,
+      "description": "Prevent externally resolved TypeScript files from being documented.",
+      "type": "boolean"
+    },
+    "excludeNotExported": {
+      "description": "Prevent symbols that are not exported from being documented.",
+      "type": "boolean",
+      "default": false
+    },
+    "excludePrivate": {
+      "description": "Ignores private variables and methods",
+      "type": "boolean",
+      "default": false
+    },
+    "excludeProtected": {
+      "description": "Ignores protected variables and methods",
+      "default": false,
+      "type": "boolean"
+    },
+    "externalPattern": {
+      "description": "Define a pattern for files that should be considered being external.",
+      "type": "string"
+    },
+    "gaID": {
+      "description": "Set the Google Analytics tracking ID and activate tracking code.",
+      "type": "string"
+    },
+    "gaSite": {
+      "default": "auto",
+      "description": "Set the site name for Google Analytics.",
+      "type": "string"
+    },
+    "gitRevision": {
+      "description": "Use specified revision or branch instead of the last revision for linking to GitHub source files.",
+      "type": "string"
+    },
+    "hideGenerator": {
+      "default": "false",
+      "description": "Do not print the TypeDoc link at the end of the page.",
+      "type": "boolean"
+    },
+    "ignoreCompilerErrors": {
+      "default": false,
+      "description": "Generates documentation, even if the project does not TypeScript compile.",
+      "type": "boolean"
+    },
+    "includeDeclarations": {
+      "default": false,
+      "description": "Turn on parsing of .d.ts declaration files.",
+      "type": "boolean"
+    },
+    "includes": {
+      "description": "Specifies the location to look for included documents (use [[include:FILENAME]] in comments).",
+      "type": "string"
+    },
+    "json": {
+      "description": "Specifies the location to output a JSON file containing all of the reflection data.",
+      "type": "string"
+    },
+    "listInvalidSymbolLinks": {
+      "description": "Emits a list of broken symbol [[navigation]] links after documentation generation",
+      "type": "boolean",
+      "default": false
+    },
+    "logger": {
+      "description": "Specify the logger that should be used.",
+      "oneOf": [{ "enum": ["none"] }, { "enum": ["console"] }]
+    },
+    "media": {
+      "description": "Specifies the location with media files that should be copied to the output directory.",
+      "type": "string"
+    },
+    "mode": {
+      "default": "modules",
+      "description": "Specifies the output mode the project is used to be compiled with.",
+      "oneOf": [{ "enum": ["file"] }, { "enum": ["modules"] }]
+    },
+    "name": {
+      "description": "Set the name of the project that will be used in the header of the template."
+    },
+    "out": {
+      "default": "docs",
```

Comment:
There is no default value for `out` - if it isn't provided documentation will not be generated (it is on the list to change this behavior to the specified default, but it hasn't been done yet)